cmd/admin-handlers.go

```diff
@@ -35,17 +35,19 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
-
 	"github.com/minio/minio/cmd/config"
 	"github.com/minio/minio/cmd/crypto"
 	xhttp "github.com/minio/minio/cmd/http"
 	"github.com/minio/minio/cmd/logger"
 	"github.com/minio/minio/cmd/logger/message/log"
 	"github.com/minio/minio/pkg/auth"
+	bandwidth "github.com/minio/minio/pkg/bandwidth"
+	b "github.com/minio/minio/pkg/bucket/bandwidth"
```

Comment:
I now see why there was bandwidth and bandwidth2. Eating up b for import isn't really good. Maybe
```suggestion
	bandwidth "github.com/minio/minio/pkg/bandwidth"
	bbandwidth  "github.com/minio/minio/pkg/bucket/bandwidth"
```

Reply:
Will change it, going with `bucketBandwidth`